main.py

```diff
@@ -91,6 +91,7 @@ def label(self):
         # Clear and redraw the text surface
         self.textSurf = self.font.render(str(displayText), 1, (86, 176, 91))
         self.surf = pygame.Surface(displaySize, pygame.SRCALPHA)
+        pygame.draw.rect(self.surf, (86, 176, 91), (0, 7.5, 5, 5))
```

Comment:
Added the code to redraw the square here. It want making any changes when you add or remove line 59 as it was then immediately cleared anyway.